.changeset/strange-ducks-rhyme.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': minor
```

Comment:
Hold up, this should be patch and contain a bit more guidance

Reply:
Hm, seemed ok to me but you are right about patch. @timbonicus mind fixing up the changeset itself in a separate PR?